04_verify.sh

```diff
@@ -196,6 +196,16 @@ EXPTD_V1ALPHA2_CRDS="clusters.cluster.x-k8s.io \
   baremetalhosts.metal3.io \
   baremetalmachines.infrastructure.cluster.x-k8s.io \
   baremetalmachinetemplates.infrastructure.cluster.x-k8s.io"
+EXPTD_V1ALPHA3_CRDS="clusters.cluster.x-k8s.io \
```

Comment:
I am now thinking is this the correct way or should we have two sets one for common CRDs and one for extra CRDs which are version specific, otherwise this list will be exhaustive? Just a suggestion

Reply:
good point, done